db/dataGenerator.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+const faker = require('faker'); // GENERATE 100 RECORDS!
+const fs = require('file-system');
+const s3 = require('./s3.js');
+
+// TOP DATA [recipeName, thumbs, description, imageURL, photoBy]
+
+let top = 1;
+const topData = [];
+
+while (top <= 100) {
+  topData.push(`
+  {
+    "recipeID": ${top},
+    "name": "${faker.random.words()}",
+    "thumbs": ${Math.ceil(Math.random() * (100 - 54) + 54)},
```

Comment:
```suggestion
    "thumbs": ${faker.random.number({ min: 25, max: 100})}
```
faker.random.number might be easier to use here

Reply:
cool nice catch, I had just used Math cause thats what I already knew, I like the uniform way though